### ISSUE TITLE
Cannot open include file: 'unistd.h'

### ISSUE THREAD

**Issue by wim07101993** [tuxedocomputers/tuxedo-control-center#76]
HI,

I am trying to build the control center for Windows but get the error:
`C:\Users\wimva\source\repos\tuxedo-control-center\src\native-lib\tuxedo_io_lib\tuxedo_io_api.hh(22,10): fatal error C10
83: Cannot open include file: 'unistd.h': No such file or directory [tuxedo-control-center\
build\TuxedoIOAPI.vcxproj]`
Which is kind of obvious since uni is for Unix... 

Is there a way to build or download the control center for Windows?

**Comment by sevillaarvin** [comment]
Does the latest release fix the issue?

**Comment by wim07101993** [comment]
I just pulled the repo and ran `npm install`. The error changed: `C:\Users\wimva\source\repos\tuxedo-control-center\src\native-lib\tuxedo_io_napi.cc(22,10): fatal error C1083: Cannot open include file: 'libudev.h': No such file or directory C:\Users\wimva\source\repos\tuxedo-control-center\build\TuxedoIOAPI.vcxproj]`.

**Comment by tuxedoxt** [comment]
Right, you need libudev nowadays as well. Usually available from distro repos.

**Comment by tuxedoder** [comment]
We only build `tcc` on linux.
